index.html

```diff
@@ -0,0 +1,177 @@
+            from different locations who will share their experience and knowledge.</p>
+        </div>
+        <div>
+          <p>2023.04.01(Sat) ~ 04(Tue)</p>
+        </div>
+        <div>
+          <p>Expo Guadalajara. Av. Mariano Otero 1499, Verde Valle, 44550 Guadalajara, Jal. Mexico.</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="program">
+      <h2 class="title">Main Program</h2>
+      <hr class="div-line">
+      <div class="card-container">
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/star_border_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Main Stage</h3>
+          <p class="program-card-description">Listen to the speakers from various countries about the messenges of
+            sharing and opening.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/business_center_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Business Land</h3>
+          <p class="program-card-description">In this land you will be able to explore new trends, acquire tools and
+            discuss innovative strategies to
+            expand your business.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/code_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Developer Land</h3>
+          <p class="program-card-description">In this scenario you will be able to discuss and discover the potential
+            of AI.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/sports_esports_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Gamer Land</h3>
+          <p class="program-card-description">Dive into the vast playing field this land has to offer and reveal its
+            secrets as you win a game to the
+            future.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/keyboard_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Workshops</h3>
+          <p class="program-card-description">Try creating your own work using open source license rather than just
+            watching at it.</p>
+        </article>
+      </div>
+      <a class="display display-none" href="#">SEE THE WHOLE PROGRAM</a>
+      <div class="join-button">
+        <p>Sign up to Jalisco Talent Land</p>
+      </div>
+    </section>
+
+    <section id="speakers" class="speakers">
+
+      <div id="flexcards" class="flex-card">
+      </div>
+
+      <div class="arrow_down">
+        <p>MORE</span></p>
```

Comment:
- [ ] like the previous reviever noted, try as much as possible to follow the design. remember you were suppose to follow the design and just personalize the content.  kindly put the `more `button under the speakers like in the original design
yours
![Screenshot (990)](https://user-images.githubusercontent.com/65298048/233718923-250b8612-5b11-4bd7-b7d2-ec7bc8fb30dc.png)
original version
![Screenshot (991)](https://user-images.githubusercontent.com/65298048/233719005-9baa5f79-3139-4271-bc88-6e65454a8735.png)